zipkin-cassandra-core/src/main/java/org/twitter/zipkin/storage/cassandra/Repository.java

```diff
@@ -848,7 +831,7 @@ private int durationIndexBucket(long ts) {
                 .setString("service_name", serviceName)
                 .setString("span_name", spanName)
                 .setInt("bucket", durationIndexBucket(timestamp))
-                .setLong("ts", timestamp)
+                .setBytesUnsafe("ts", serializeTs(timestamp))
```

Comment:
do we still need to use this workaround now that we're writing millis – we should be able to just do `.setDate`? Unless we're just trying to avoid allocating the Date object.